homeassistant/components/bluetooth/update_coordinator.py

```diff
@@ -84,17 +112,9 @@ def _async_stop(self) -> None:
             self._cancel_track_unavailable = None
 
     @callback
-    def _async_handle_unavailable(self, address: str) -> None:
-        """Handle the device going unavailable."""
-        self._present = False
-
-    @callback
-    def _async_handle_bluetooth_event(
```

Comment:
One of the goal is to eliminate `_async_handle_bluetooth_event` here since in the future we won't be able to count on setting `self._present` or `self.last_seen` here because not all scanner will provide RSSI updates